src/coreclr/vm/prestub.cpp

```diff
@@ -2055,6 +2057,7 @@ extern "C" void* STDCALL ExecuteInterpretedMethod(TransitionBlock* pTransitionBl
     frames.interpMethodContextFrame.startIp = dac_cast<PTR_InterpByteCodeStart>(byteCodeAddr);
     frames.interpMethodContextFrame.pStack = sp;
     frames.interpMethodContextFrame.pRetVal = (retBuff != NULL) ? (int8_t*)retBuff : sp;
+    frames.interpMethodContextFrame.hiddenArgument = GetMostRecentUMEntryThunkData();
```

Comment:
Should we have a interpreter opcode to save UMEntryThunkData* that we emit as the first thing in the IL stub so that we do not have to do it every time here? `CORJIT_FLAG_PUBLISH_SECRET_PARAM` is the flag that the VM passes to the JIT when the method uses the hidden arg.

(